wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandTestBase.java

```diff
@@ -1,5 +1,5 @@
 /*----------------------------------------------------------------------------*/
-/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Copyright (c) 2018-2020 FIRST. All Rights Reserved.                        */
```

Comment:
Can you please put these changes in a separate PR?  Thanks.

Reply:
PR opened: #2472